WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -170,6 +171,9 @@ public WordPressDB(Context ctx) {
             case 64:
                 // add site icon
                 mDb.execSQL(SiteSettingsModel.ADD_SITE_ICON);
+            case 65:
+                // add external users only to publicize services table
+                mDb.execSQL(PublicizeTable.ADD_EXTERNAL_USERS_ONLY);
```

Comment:
This crashes when there is no pre-existent `PublicizeTable`. Apparently, it is only created when `PublicizeListActivity` is [first visited here](https://github.com/wordpress-mobile/WordPress-Android/blob/fix/facebook-publicize-aug-1/WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeListActivity.java#L72) . We should add a check that the table exists before creating it, or force creating it before this step execution by calling `PublicizeTable.createTables(mDb);` right above this `ALTER TABLE` statement.